pkg/watcher/internal/metricsprovider/signalfx.go

```diff
@@ -422,3 +430,13 @@ func getMetricsFromPayloads(metricData interface{}, metadata interface{}) (map[s
 
 	return hostMetricMap, nil
 }
+
+// This function extracts host name from its FQDN
+// This assumes that host names themselves don't contain "."
+func cleanUpHostname(hostname string) string {
```

Comment:
Can you give an example of the input and output expected by this function? (It may also be worth putting this in as a comment.)

Reply:
Added, also modified var/function names to make it clearer